inf-clojure.el

```diff
@@ -238,11 +238,23 @@ This should usually be a combination of `inf-clojure-prompt' and
 `inf-clojure-subprompt'."
   :type 'regexp)
 
-(defcustom inf-clojure-prompt-on-set-ns t
+(defcustom inf-clojure-prompt-on-commands t
   "Controls whether to prompt when switching namespace."
   :type '(choice (const :tag "always" t)
                  (const :tag "never" nil))
-  :package-version '(inf-clojure . "2.0.0"))
+  :package-version '(inf-clojure . "2.0.0")
+  :safe 'booleanp)
+
+(defun inf-clojure--prompt-on-commands-p (&optional arg)
```

Comment:
Agree on this! I will revert back to the previous version and abolish the defcustom.